blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/UserImpl.java

```diff
@@ -36,15 +44,20 @@
 
     protected final User user;
 
+    private final BlueOrganization organization;
+    private final AccessControlled organizationBase;
+
     private final Reachable parent;
-    public UserImpl(User user, Reachable parent) {
+
+    public UserImpl(@Nonnull BlueOrganization organization, @Nonnull User user, Reachable parent) {
```

Comment:
Didn't see you second comment when replying, that's right this object it's only to return JSON so not sure that would be possible